applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -232,6 +232,8 @@ config WALKIE_TALKIE_DEMO
 
```

Comment:
Remember OCT-2440 in the commit message.

Reply:
Done